tests/login/ready.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Positive login test for local"
```

Comment:
Other tests are using METHOD="${METHODS:- local container}" and let adjust to set it to contain also virtual.
I think we can't us artemis in the test suite (needs API token). Also tmt run created in the test suite should be removed after test finishes. Check other tests for the 'for METHOD ...` inspiration.

Reply:
Sure, added these lines to main.fmf.
```
 /ready:
     summary: Try to log into guest to verify property is_ready
     test: ./ready.sh
+    environment:
+        METHODS: local container virtual
```

Reply:
Test for artemis is removed.